CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- Continuously track all URLs in view
```

Comment:
If there's no change for the user, we shouldn't add a change log entry.

I don't think this fixes any issues with URLs updating below the cursor yet, right? So we probably shouldn't add any change log entry if nothing changed for the user.

Reply:
No, this doesn't fix those issues, although that would be a lot easier to fix with this change. However, fixing the URL update issues will probably require a lot of changes to `input.rs` and `update.rs` - I'm totally happy to work on that, but should I wait until the glutin rework lands first?

Reply:
Probably a good idea, yeah. I think I'm going to try to land the glutin rework for initial testing this weekend though.